src/main/java/pages/LoginPage.java

```diff
@@ -0,0 +1,30 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class LoginPage extends BasePage {
+
+    public LoginPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public static final By USERNAME_INPUT = By.xpath("//*[@data-test='username']");
+    public static final By PASSWORD_INPUT = By.xpath("//*[@data-test='password']");
+    public static final By LOGIN_BUTTON = By.xpath("//*[@id='login-button']");
+    public static final By ERROR_MESSAGE_PAGE_LOGIN = By.xpath("//*[@class ='error-message-container error']");
```

Comment:
ERROR_MESSAGE_LOCATOR
By.xpath("//*[contains(@class, 'error-message-container') and contains(@class, 'error')]")